applications/order/groovyScripts/entry/cart/LookupBulkAddSupplierProducts.groovy

```diff
@@ -62,22 +60,23 @@ conditionList = []
 
 if (productId) {
     // make sure the look up is case insensitive
-    conditionList.add(EntityCondition.makeCondition(EntityFunction.upper(EntityFieldValue.makeFieldValue("productId")),
-                                     EntityOperator.LIKE, productId.toUpperCase() + "%"))
+    conditionList.add(EntityCondition.makeCondition(EntityFunction.upper(EntityFieldValue.makeFieldValue('productId')),
+                                     EntityOperator.LIKE, productId.toUpperCase() + '%'))
 }
-if (!supplier) {
-    supplierPartyId = shoppingCart.getOrderPartyId()
+if (supplier) {
+    supplierPartyId = supplier.getString('partyId')
 } else {
-    supplierPartyId = supplier.getString("partyId")
+    supplierPartyId = shoppingCart.getOrderPartyId()
```

Comment:
This catch by the rules **InvertedIfElse** rule can be also converted with the rule **CouldBeElvis** rule by : 
```
supplierPartyId = supplier?.partyId ?: shoppingCart.getOrderPartyId()
```